src/app/visual/flyout/flyout-visual.component.ts

```diff
@@ -1,8 +1,14 @@
 import {
-  Component
+  Component,
+  OnDestroy
 } from '@angular/core';
 
 import {
+  Subject
+} from 'rxjs';
```

Comment:
Use `'rxjs/Subject'` until we can update to the latest version of RxJS (they've improved their imports in later versions, and don't pull in more than they ought to).

Reply:
done.